egendata/infrastructure
# chore: 🤖 test environment

https://github.com/egendata/infrastructure/blob/ca2b826abd4b8986b2c34c71bee70b275185adb2/openshift/README.md#release-a-new-tag-to-test

### How to test:
- I have manually updated the deployment config and set an image tag that is older (like 0.30.1 for operator), then I apply the file doing `oc apply -f openshift/test/operator-DeploymentConfig.yml` and finally I commit this change to be able to see that the commit step of the script also works
- I make sure everything is updated in Openshift (you can view the image that runs here for example https://console.dev.services.jtech.se:8443/console/project/mydata/browse/dc/operator-test?tab=configuration)
- then I source the script and run `deploy_operator 0.30.3` and I should see a commit (`chore: `) and then if I check in Openshift again ☝️ I should see that it's running the new image
- other scenarios I've tested is using a tag that doesn't exist, older tag or same tag
